src/run.rs

```diff
@@ -23,10 +23,11 @@ enum BuildResult {
 fn build(
     progress: &mut ConsoleProgress,
     parallelism: usize,
+    build_filename: &str,
     regen: bool,
     target_names: &[String],
 ) -> anyhow::Result<BuildResult> {
-    let mut state = trace::scope("load::read", load::read)?;
+    let mut state = trace::scope_str("load::read", load::read, build_filename)?;
```

Comment:
This can be
`trace::scope("load::read", || load::read(build_filename))?;`

Reply:
done